public/locales/fr/email.json

```diff
@@ -1,32 +1,32 @@
 {
-  "header": "Obtenir mon numéro de dossier",
+  "header": "Obtenir le numéro de dossier de ma demande de passeport",
   "description": "Remplissez les champs ci-dessous pour obtenir votre numéro de dossier. Assurez-vous que vos informations correspondent à votre formulaire de demande de passeport. Nous enverrons ensuite le numéro de dossier par courriel à l'adresse courriel enregistrée.",
   "email": {
-    "label": "Adresse courriel",
+    "label": "Courrier électronique",
```

Comment:
It needs to be "Adresse courriel". 1. because it's the address to of the email's recipient and 2. because it's the exact term used in the application form.

![image](https://user-images.githubusercontent.com/114004123/210595405-cbabf3fa-ca3a-469a-91b6-280c3afd2f7c.png)

Reply:
`Email address` is good in english.

`Email` represent the message that translate to `Courriel` in french and `Address` is the address of the email's recipient.

![image](https://user-images.githubusercontent.com/114004123/210595768-71e73219-1639-4448-88b6-7a24b385a095.png)